packages/env-tests/src/tests/attestation.ts

```diff
@@ -0,0 +1,137 @@
+import { StableTokenWrapper } from '@celo/contractkit/lib/wrappers/StableTokenWrapper'
+import { generateKeys, generateMnemonic } from '@celo/utils/lib/account'
+import { privateKeyToAddress } from '@celo/utils/lib/address'
+import { sleep } from '@celo/utils/lib/async'
+import { describe, test } from '@jest/globals'
+import BigNumber from 'bignumber.js'
+import Logger from 'bunyan'
+import twilio from 'twilio'
+import { Context } from '../context'
+import { envVar, fetchEnv } from '../env'
+import { getKey, ONE, TestAccounts } from '../scaffold'
+import {
+  getIdentifierAndPepper,
+  getPhoneNumber,
+  pollForMessagesAndCompleteAttestations,
+  reportErrors,
+  requestAttestationsFromIssuers,
+  requestMoreAttestations,
+} from '../shared/attestation'
+
+async function fundAttestationAccount(
+  context: Context,
+  stableToken: StableTokenWrapper,
+  root: string,
+  from: string,
+  logger: Logger
+) {
+  context.kit.defaultFeeCurrency = stableToken.address
+
+  const rootTransfer = stableToken.transfer(from, ONE.toString())
+  const rootTransferReceipt = await rootTransfer.sendAndWaitForReceipt({ from: root })
+
+  logger.debug(
+    {
+      receipt: rootTransferReceipt,
+      account: from,
+    },
+    'funded attestation account'
+  )
+}
+
+export function runAttestationTest(
+  context: Context,
+  attetationsToRequest = 3,
+  configuredPepper = 'HARDCODED_PEPPER',
+  odisContext = 'mainnet'
+) {
+  describe('Attestation Test', () => {
+    const logger = context.logger.child({ test: 'attestation' })
+
+    // TODO: Check for TWILIO ENV vars
+    test('receive attestations for a phone number', async () => {
+      const root = await getKey(context.mnemonic, TestAccounts.Root)
+      context.kit.addAccount(root.privateKey)
+
+      const mnemonic = await generateMnemonic()
+      const from = await generateKeys(mnemonic)
```

Comment:
this is slow
do we need a new one everytime?

Reply:
Do you know how slow SMS sends are :P

Since the effects of the attestation tests persists between invocations, I wanted to avoid a scenario in which a identifier/account combo already exhausted all issuers. Thus the easiest thing is to just create a new account each time.